File: evo.funders.commonmessages/v1/DotNet/Request/CancelApplicationRequest.cs
using AzureFunderCommonMessages.DotNet.Request.DataTypes;
using Newtonsoft.Json;

namespace AzureFunderCommonMessages.DotNet.Request
{
    public class CancelApplicationRequest : BaseCommonRequest
    {
        public CancelApplicationRequest()
        {
            Data = new CancelApplicationRequestData();
        }

        [JsonProperty("data", Required = Required.Always)]
        public new CancelApplicationRequestData Data { get; set; }
    }
}

File: evo.funders.commonmessages/v1/PHP/src/Request/CancelApplicationRequest.php
<?php


namespace AzureFunderCommonMessages\PHP\Request;

use AzureFunderCommonMessages\PHP\Traits\JsonSerializer;

class CancelApplicationRequest extends AbstractData implements \JsonSerializable
{
    use JsonSerializer;

    private ?string $funderApplicationId;

    /**
     * @return string|null
     */
    public function getFunderApplicationId(): ?string
    {
        return $this->funderApplicationId;
    }

    /**
     * @param string|null $funderApplicationId
     * @return CancelApplicationRequest
     */
    public function setFunderApplicationId(?string $funderApplicationId): CancelApplicationRequest
    {
        $this->funderApplicationId = $funderApplicationId;
        return $this;
    }

    public static function fromJson(array $data): self
    {
        $instance = new self();
        $instance->funderApplicationId = $data["funderApplicationId"];
    }

}